services/QuillLMS/app/views/pages/home_new.html.erb

```diff
@@ -20,7 +20,7 @@
   </div>
   <div class="q-hero-footer">
     <p class="text-white">
-      <span class="hero-footer-underline"><%= number_to_human(@number_of_students).downcase %></span> students have written <span class="hero-footer-underline"><%= number_to_human(@number_of_sentences).downcase %></span> sentences on Quill.
```

Comment:
number_to_human throws an exception after the rails bump, due to the internally bundled BigDecimal library's API changing. Since we're only using `number_to_human` in this one file, I just wrote a custom helper to replace `number_to_human`.

Reply:
Fixed it, thanks

Reply:
Also, what is the specific exception that you get?

Reply:
`undefined method `new' for BigDecimal:Class`